packages/manager/modules/pci/src/projects/project/failover-ips/order/agora/order.html

```diff
@@ -0,0 +1,41 @@
+<oui-modal
```

Comment:
In `packages/manager/modules/pci/src/projects/project/failover-ips/order/legacy/order.html` template the `<oui-modal>` is wrap with a `<form>` element.

Shouldn't be the same from this template?

Reply:
Indeed